x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_right_panel.cy.ts

```diff
@@ -61,13 +61,18 @@ import { getNewRule } from '../../../../objects/rule';
 import { ALERTS_URL } from '../../../../urls/navigation';
 import { waitForAlertsToPopulate } from '../../../../tasks/create_new_rule';
 import { TOASTER } from '../../../../screens/alerts_detection_rules';
+import { ELASTICSEARCH_USERNAME, IS_SERVERLESS } from '../../../../env_var_names_constants';
+
+// We need to use the 'soc_manager' role in order to have the 'Respond' action displayed in serverless
```

Comment:
Thanks for making this change. Is there is list of serverless roles somewhere?

Reply:
> Thanks for making this change. Is there is list of serverless roles somewhere?

Yes @logeekal you can find the security-specific [here](https://github.com/elastic/kibana/blob/main/packages/kbn-es/src/serverless_resources/security_roles.json).